manual/installation.tex

```diff
@@ -632,10 +610,9 @@ \subsubsection{Building QMCPACK}
 module load cuda
 mkdir build_summitdev
 cd build_summitdev
-cmake -DCMAKE_C_COMPILER="mpicc" \
-      -DCMAKE_C_FLAGS="-std=gnu11 -qsmp -O3" \
-      -DCMAKE_CXX_COMPILER="mpicxx" \
-      -DCMAKE_CXX_FLAGS="-std=gnu++11 -qnoxlcompatmacros -qsmp -O3" \
+cmake -DCMAKE_C_COMPILER="mpixlc" \
+      -DCMAKE_CXX_COMPILER="mpixlC" \
+      -DBUILD_LMYENGINE_INTERFACE=0 \
```

Comment:
You are technically correct (the best kind of correct) ;-)

I left it in for those folks who want to compile the CPU version in which case the CXX11 test now passes and formic/utils MPI implementation causes a hard compile error as the C++ bindings are no longer available on SummitDev.